nengo/utils/graphs.py

```diff
@@ -98,6 +98,43 @@ def toposort(edges):
     return ordered
 
 
+def transitive_closure(edges, topo_sorted=None):
+    """Constructs the transitive_closure of a DAG.
```

Comment:
According to the [first link I found by searching](http://www.geeksforgeeks.org/transitive-closure-of-a-graph/), I can consider a "transitive closure" as a synonym for a "reach-ability matrix". Is that accurate? Or is something else going on here? If yes, could you elaborate? If no, would it be a good idea to insert that layman's terms translation?